statsmodels/discrete/discrete_model.py

```diff
@@ -2073,28 +2069,32 @@ def _score_nbin(self, params, Q=0):
             alpha = params[-1]
         params = params[:-1]
         exog = self.exog
-        y = self.endog[:,None]
-        mu = self.predict(params)[:,None]
+        y = self.endog[:, None]
+        mu = self.predict(params)[:, None]
         a1 = 1/alpha * mu**Q
+        # Note: Q is either 0 or 1, so a1 is either 1/alpha or mu/alpha.
         if Q: # nb1
-            dparams = exog*mu/alpha*(np.log(1/(alpha + 1)) +
-                       special.digamma(y + mu/alpha) -
-                       special.digamma(mu/alpha))
+            digamma_part = (
+                        special.digamma(y+mu/alpha)
```

Comment:
we follow old pep-8 with spaces around all operators, `+, -, *, /, ...` except `**` (power)